ios/atb/Info.plist

```diff
@@ -32,6 +32,14 @@
 				<string>atb</string>
 			</array>
 		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>com.googleusercontent.apps.827196677776-fft65c0v4jp8u7fdlqo03ep1u0n66fhf</string>
+			</array>
+		</dict>
```

Comment:
Where does this come in?

Reply:
It's for setting up recaptcha verification on iOS. Described here:
https://firebase.google.com/docs/auth/ios/phone-auth#set-up-recaptcha-verification

Reply:
Aha. That value seems to be unique per bundle identifier (`debug`, `staging`, `store`), so we'll probably need to override it per environment then too: https://github.com/AtB-AS/mittatb-app/blob/8b249184bdf8c14ec15b1a4b92559dba77e83af7/scripts/override-config-files.sh#L16

Reply:
On iOS, we have a unique `GoogleService-Info.plist` per env (see `ios`-folder, as well as each folder under `env`). So we can probably extract `REVERSED_CLIENT_ID` from those, then set it in our `Info.plist` as with the other API-keys, using `PlistBuddy`

Reply:
Fixed